.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 
-files: "nilearn/de.*"
+files: "(nilearn/de.*)|(.*binder/)|(.*circleci/)|(.*github/)|(maint_tools/.*)|(setup.*)|(.*coverage)|(.*toml$)|(.*in$)|(.*yml)|(.*yaml)|(CONTRIBUTING.rst)|(AUTHORS.rst)|(LICENSE)|(Makefile)|(.*gitignore)"
```

Comment:
this regular expression will get uglier and uglier as we expand the scope of pre-commit, but it will eventually get better once we can just apply pre-commit to the whole repo.